src/render_tasks/d3d12_rtao_task.hpp

```diff
@@ -16,6 +16,13 @@
 
 namespace wr
 {
+	namespace rtaoSettings
+	{
+		static float bias = 0.01f;
```

Comment:
We have a proper way to deal with settings. See hbao or ansel.

Reply:
Settings now use the proper system!